python/paddle/fluid/incubate/fleet/parameter_server/__init__.py

```diff
@@ -123,18 +123,25 @@ def init_pserver(self):
             print("You should run DistributedOptimizer.minimize() first")
             sys.exit(-1)
 
-    def init_worker(self, programs):
+    def init_worker(self, programs, scopes=None):
```

Comment:
It seems that a user has to specify a scope for each program. Maybe you can add the scope into python program class in the future.